partiql/Cargo.toml

```diff
@@ -46,7 +46,7 @@ time = { version = "0.3", features = ["macros"] }
 criterion = "0.5"
 rand = "0.8"
 
-assert_matches = "1.5"
+assert_matches = "1"
```

Comment:
Just to double-check: Do we want to move from `1.5` to `1`?

Reply:
It's what all other crates use: 
```
13:50❯ rg assert_matches -g "*.toml"
partiql-eval/Cargo.toml
35:assert_matches = "1"

partiql/Cargo.toml
49:assert_matches = "1"

partiql-logical-planner/Cargo.toml
42:assert_matches = "1"

partiql-parser/Cargo.toml
50:assert_matches = "1"
```